presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -6731,6 +6734,23 @@ public void testTimeLiterals()
     }
 
     @Test
+    public void testArrayShuffle()
+            throws Exception
+    {
+        int equalCount = 0;
+        int testCount = 23;
+        List<Integer> expected = asList(2, 8, 7, 5, 4, 6, 3);
+        for (int i = 0; i < testCount; i++) {
+            List<Integer> result = (List<Integer>) computeActual("SELECT shuffle(array[2, 8, 7, 5, 4, 6, 3])").getOnlyValue();
+            assertEqualsIgnoreOrder(result, expected);
+            if (result.equals(expected)) {
+                equalCount++;
+            }
+        }
+        assertTrue(equalCount < 3);
```

Comment:
Given the number of elements in the array, the chances of getting the same result are tiny. Even a single equality is unlikely.

The test should validate that multiple executions produce different results, not just that they are not equal to the "expected" list.